src/libs/nuwebcomm-scripts.js

```diff
@@ -216,43 +216,44 @@ $(document).ready(function() {
     switchTab2($(this));
   });
 
+  // REMOVING JS SIDE AS WE IMPLEMENT CSS-BASED TRANSITION
   // photo feature flip image on hover
-  $(".photo-feature")
-    .has(".back")
-    .hover(
-      function() {
-        flipCard($(this));
-      },
-      function() {
-        unflipCard($(this));
-      }
-    );
-  $(".photo-feature")
-    .has(".back")
-    .click(function(e) {
-      if (
-        $(this)
-          .find(".front")
-          .is(":visible")
-      ) {
-        flipCard($(this)); // hover was not triggered, so flip on click
-        e.preventDefault();
-      } // else goodbye!
-    });
-  function flipCard(cardObj) {
-    var img_height = cardObj.find("img").css("height");
-    $(".front").height(img_height);
-    $(".back").height(img_height);
-    cardObj.find(".front").hide();
-    cardObj.find(".back").show();
-  }
-  function unflipCard(cardObj) {
-    var img_height = cardObj.find("img").css("height");
-    $(".front").height("auto");
-    $(".back").height("auto");
-    cardObj.find(".front").show();
-    cardObj.find(".back").hide();
-  }
+  // $(".photo-feature")
+  //   .has(".back")
+  //   .hover(
+  //     function() {
+  //       flipCard($(this));
+  //     },
+  //     function() {
+  //       unflipCard($(this));
+  //     }
+  //   );
+  // $(".photo-feature")
```

Comment:
@katdivyareddy10 Can we include this piece of the logic (lines 231-242) also in `PhotoFeature` component?   This logic is for touch/mobile devices.  

If you look at https://www.northwestern.edu/templates/page-types/full-width/modules.html# on your phone and/or tablet you can see the behavior we want to achieve.